lib/mappers/michigan_mapper.py

```diff
@@ -307,4 +318,17 @@ def map_multiple_fields(self):
         self.map_collection()
         self.map_date_and_publisher()
         self.map_data_provider_and_intermediate_provider()
-        self.map_object_and_is_shown_at()
\ No newline at end of file
+        self.map_object_and_is_shown_at()
+
+    # Get values from property
+    # Returns list of values
+    def get_date(self, s):
+        dates = []
+
+        if not isinstance(getprop(s, "dateIssued"), basestring):
+            for d in iterify(getprop(s, "dateIssued")):
+                dates.append(getprop(d, "#text"))
```

Comment:
@markbreedlove I made of note of this in the mapper, I think the refactor might be a bit more involved that I want to bite off atm (in terms of re-testing the mapping). The unfinished business that we are going to ignore for their initial ingest with MI is about the mapping of dates so I'd like to propose using that ticket as the opportunity for a better refactor of this block since this is working now.

Thoughts?